test/browser/wrapper.jsx

```diff
@@ -0,0 +1,181 @@
+/* global before, beforeEach, after, describe, expect, it */
+import Wrapper from '../../src/wrapper'
+
+const injectSelectToEnhanceIntoDOM = (element, id, name, options, selectedOption) => {
+  var $select = document.createElement('select')
+  if (id) {
+    $select.id = id
+  }
+  if (name) {
+    $select.name = name
+  }
+
+  for (var optionKey in options) {
```

Comment:
Only if you want™:

You can switch this to `Object.keys(options).map(foo).forEach(bar)` which separates the array transformation logic from the `appendChild` side effect.